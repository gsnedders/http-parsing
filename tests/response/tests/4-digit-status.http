HTTP/1.1 1234 OK
Connection: Close

Test