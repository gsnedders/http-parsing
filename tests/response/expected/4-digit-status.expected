{
	"HTTP": {
		"version": "1.1",
		"status": 1234,
		"reason": "OK"
	},
	
	"headers": {
		"connection": "close"
	},
	
	"body": "Test"
}